doxybindgen.toml

```diff
@@ -189,6 +224,29 @@ blocklist = [
     # Didn't support
     'Scale',
 ]
+[types.wxSizer]
+blocklist = [
+    # virtual
+    'wxSizer',
+    # maybe binding bug, return type didn't generated
+    'GetChildren',
+    # deprecated
+    'GetChildren1',
+    # deprecated, use Remove() without args
+    'Remove',
+    # deprecated, use FitInside()
+    'SetVirtualSizeHints',
```

Comment:
i can't tell why deprecated but not removed method not callable. revisit them later.

Reply:
* Remove - actually is absent in header file.
* SetVirtualSizeHints - declated only when WXWIN_COMPATIBILITY_2_8 is defined.